vowpalwabbit/example.h

```diff
@@ -49,10 +52,12 @@ typedef union
   float scalar;
   v_array<float> scalars;           // a sequence of scalar predictions
   ACTION_SCORE::action_scores a_s;  // a sequence of classes with scores.  Also used for probabilities.
+  VW::actions_pdf::pdf prob_dist;
   VW::decision_scores_t decision_scores;
   uint32_t multiclass;
   MULTILABEL::labels multilabels;
   float prob;  // for --probabilities --csoaa_ldf=mc
+  VW::actions_pdf::action_pdf_value a_pdf;
```

Comment:
I think we should opt for a more descriptive name here

Reply:
Suggestions?

Reply:
I don't really know what this is, which is the main reason why I called it out. Additionally it is unclear to me what `action_pdf_value` and how it differs to an `action_pdf`

Reply:
Changed